apps/demos/src/WalletSelector/SignAndSendTransaction.tsx

```diff
@@ -0,0 +1,43 @@
+import { signAndSendTransaction } from '@bos-web-engine/wallet-selector-plugin';
+import { useCallback, useState } from 'react';
+
+export default function () {
+  const [transferAmount, setTransferAmount] = useState('');
+  const [receiverId, setReceiverId] = useState('');
+  const [transaction, setTransaction] = useState('');
+
+  const signAndSend = useCallback(async () => {
+    const { transaction } = await signAndSendTransaction({
+      actions: [
+        {
+          type: 'Transfer',
+          params: {
+            deposit: transferAmount,
+          },
+        },
+      ],
+      receiverId,
+    });
+    setTransaction(JSON.stringify(transaction, null, 2));
+  }, [transferAmount, receiverId]);
```

Comment:
This is pretty nit picky, but this looks like a situation where you don't really need to use `useCallback()` since you don't have any effects that require this function in a dependency array. This would simply be:

```ts
const signAndSend = async () => {
    ...
  };
```